asciidoc/sdpi-supplement-issues.adoc

```diff
@@ -7,10 +7,18 @@
 === SDPi "Topic of Interest" Issue Management
 
 All SDPi supplement issues are tracked in the https://github.com/IHE/DEV.SDPi/issues[IHE Github DEV.SDPi repository Issues section].
-Filter the issues on "Topic of Interest" to see a full list.  +
-
-* To see the full list of OPEN issues, filter on: is:issue is:open label:"Topic of Interest"
-* To see the full list of CLOSED issues, filter on: is:issue is:closed label:"Topic of Interest"
+Filter the issues on "Topic of Interest" to see a full list of issues related to any "Topic of Interest".  +
```

Comment:
Why not adding the hyperlinks to the search queries here instead of explaining how to find them?

-  https://github.com/IHE/DEV.SDPi/issues?q=is%3Aissue+is%3Aopen+label%3A%22Topic+of+Interest%22
- https://github.com/IHE/DEV.SDPi/issues?q=is%3Aissue+is%3Aclosed+label%3A%22Topic+of+Interest%22

It's the internet after all :-)

Reply:
Or let me guess... this is one of the changes you never made but popped up in your branch after merging from master?

Reply:
Good suggestion about using hyperlinks instead.

And no, this is actually the change that this issue and PR should be about. All the other changes were dragged in from other commits made to the master during the lifetime of this branch....